### DISCUSSION TITLE
Can I autofocus not on hour input but on seconds?

### DISCUSSION THREAD

**Discussion by lofti198** [wojtekmaj/react-time-picker#149] [Q&A] [answered]
Can I autofocus not on hour input but on seconds? Like this:
![image](https://user-images.githubusercontent.com/42960488/190556698-cdc250b7-ef06-4d53-b612-be337d9dff8e.png)

**Comment by wojtekmaj** [comment, marked as answer]
Not easily, but you can:

https://codesandbox.io/s/react-time-picker-focus-second-tsctog

**Reply by lofti198** [reply]
Cool! That is what I need. By the way - can I also process Enter button (so user can show that he finished input)?
